src/app/pages/main/main.component.scss

```diff
@@ -1 +1,3 @@
-
+#main {
```

Comment:
Why not use 
```
.main {
}
```
instead of ID

Reply:
Hmm, it not make sense to me. I will use `.overflow-hidden` instead

Reply:
done :+1: